packages/yarnpkg-builder/sources/commands/build/bundle.ts

```diff
@@ -126,8 +126,6 @@ export default class BuildBundleCommand extends Command {
               'process.env.NODE_ENV': JSON.stringify(`production`),
               // For ink
               'process.env.DEV': JSON.stringify(`false`),
-              // For ourselves
-              'process.env.TEST_ENV': `false`,
```

Comment:
Could you elaborate on this change / why are we including our test code in production now?

Reply:
Because tests use the bundle too; the login tests can't work at all currently. It's also a very small amount of code, with a reasonable use case for users (enable non-official programmatic usage), so I'm not sure requiring an extra `--no-tests` flag is really worth the very small size gain (110 bytes as of https://github.com/yarnpkg/berry/pull/4315).

Reply:
The tests use the un-minified build so that shouldn't affect our tests.
It's not really about the size, it's about not breaking users (test) setup if they set `TEST_ENV` and run `yarn`.
If we want to include our test code we should at least use a more specific name than `TEST_ENV` so that users don't accidentally break their setup.

Reply:
Ah, I see - I was talking about the local tests, which I often run after a simple `yarn build:cli`. I'm fine with using a Yarn-tagged env name, will make the change 👍